src/ucp/rma/flush.c

```diff
@@ -99,32 +113,33 @@ static void ucp_ep_flush_progress(ucp_request_t *req)
                       ucs_status_string(status));
         if (status == UCS_OK) {
             req->send.flush.started_lanes |= UCS_BIT(lane);
+            ucs_assert(req->send.state.uct_comp.count > 0);
```

Comment:
can we unite checking count>0 and decrementing it to a helper function?
probably could be reused in other places too